servers/alhena/default.nix

```diff
@@ -0,0 +1,20 @@
+{ modulesPath, inputs, config, pkgs, ... }:
+let
+  vs = config.vault-secrets.secrets;
+in {
+  imports = [
+    "${modulesPath}/virtualisation/amazon-image.nix"
+    inputs.hermetic.nixosModules.hermetic
+  ];
+
+  systemd.services.amazon-init.enable = false;
```

Comment:
Already in https://github.com/serokell/gemini-infra/blob/mkaito/ops1113-hermetic/common.nix#L10